### ISSUE TITLE
Post not sent to the Slack channel 

### ISSUE THREAD

**Issue by logankilpatrick** [logankilpatrick/StackOverflowBot#8]
https://stackoverflow.com/questions/57965526/how-to-skip-errors-in-a-loop-for-julia

This question doesn't appear in the slack. Either the tag was edited, or there was a weird time issue. (edge case related to the time calculation).

**Comment by kdheepak** [comment]
If the question is posted exactly 30 minutes before the github action runs (unlikely), or as you mention if the tag is edited or added after the action runs, the current implementation will not capture it. Maybe what should happen is read the last hour of Slack channel posts by the bot and check what posts are missing from the last hour of Julia StackOverflow questions and post those; and run this script every 30 minutes or so. Thoughts?

**Comment by logankilpatrick** [comment]
Yes, that’s likely the right solution. All I would need todo is build a basic logging system. I can even check questions in the last 32 minutes, see if any of them are in my log, if not, post them. 

Best, 
Logan Kilpatrick

**Comment by logankilpatrick** [comment]
The issue ends up being that the environment gets cleaned after an action is done, so I would have to remotely commit the updated log file. 

Best, 
Logan Kilpatrick

**Comment by logankilpatrick** [comment]
Might consider using a firebase server in order to avoid the overhead of committing and such.

**Comment by logankilpatrick** [comment]
The opposite can also be true. It’s possible that there’s a post that is sent twice is posted near the top the hour. It’s happend twice.
